components/leaflet-maps/ClustersMap.vue

```diff
@@ -66,7 +66,6 @@
 
         <l-icon :icon-anchor="iconAnchor">
           <div class="icon-wrapper">
-            <div v-if="hasWikiEntry(marker)" class="icon-wiki">W</div>
```

Comment:
Have you checked this with Ewan? I'm sure you probably have, but he did ask for these in the first place. If you have, I'll merge in.